# Request
geometry_msgs/PoseStamped pick
geometry_msgs/PoseStamped place
---
# Response